packages/avatar-group/src/styles/vaadin-avatar-group-base-styles.js

```diff
@@ -3,62 +3,60 @@
  * Copyright (c) 2020 - 2025 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
+import '@vaadin/component-base/src/style-props.js';
```

Comment:
Added missing import since the component uses `--vaadin-focus-ring-width` defined in `style-props.js`.